packages/icons/processIcons.ts

```diff
@@ -0,0 +1,142 @@
+import fs from 'fs';
+import ICO from 'icojs';
+import path from 'path';
+import { sync as rimrafSync } from 'rimraf';
+import {
+  iconComponentTemplate,
+  getIconDataName,
+  getComponentName,
+} from './component-template';
+import { filterBlockedIcons } from './iconBlockList';
+
+export interface IconData {
+  filePath: string;
+  reactImportPath: string;
+  variant: string;
+  id: string;
+  width: number;
+  height: number;
+  bpp: number;
+  buffer: ArrayBuffer;
+}
+
+const ICONS_FOLDER = './src/icons';
+const ICON_DEMOS_FILE_PATH = path.join('src', 'iconDemos.ts');
+const ICONS_INDEX_FILE_PATH = path.join('src', 'index.ts');
+const REACT_FOLDER = path.join('src', 'react');
+const getReactFileName = (name: string) =>
+  path.join(REACT_FOLDER, `${name}.tsx`);
+const IMAGE_FOLDER = path.join('.', 'png');
+const getImageFileName = (imageId: string) =>
+  `${path.join(IMAGE_FOLDER, imageId)}.png`;
```

Comment:
Ahhh right. `path.relative` also returns windows paths: https://nodejs.org/api/path.html#path_path_relative_from_to

I'll change the path definitions to just use template strings instead of `path.join`